lsp-completion.el

```diff
@@ -164,10 +170,13 @@ This will help minimize popup flickering issue in `company-mode'."
 
 (defun lsp-completion--annotate (item)
   "Annotate ITEM detail."
-  (-let (((&CompletionItem :detail? :kind?) (plist-get (text-properties-at 0 item)
+  (-let (((&CompletionItem :detail? :kind? :label-details?) (plist-get (text-properties-at 0 item)
                                                        'lsp-completion-item)))
     (concat (when (and lsp-completion-show-detail detail?)
               (concat " " (s-replace "\r" "" detail?)))
+            (when (and lsp-completion-show-label-description label-details?)
+              (when-let ((description (and label-details? (gethash "description" label-details?))))
```

Comment:
you should not use gethash since when lsp-mode is compiled with lsp-use-plists = t then label-details? will be plist. You should define a structure and then use either destructuring or the generated getters

Reply:
Any example for it? I'm not very familiar with lisp. And I don't know how to change it?

Reply:
Define the structure in lsp-protocol (just like the one you edited) and then use `lsp:<struct-name>-<field-name>` as an accessor.

Reply:
@yyoncho we could try to documentate lsp-protocol.el to explain all that creates and why we should use it, it's a lsp-mode specific thing that is not that easy for new users get it

Reply:
yep, we should do that in contribution section.